jakarta.batch.arquillian.extension/src/main/java/ee/jakarta/tck/batch/arquillian/DeploymentPackageType.java

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
Should be:
> Copyright (c) 2022 Contributors to the Eclipse Foundation

Should have:
> * SPDX-License-Identifier: Apache-2.0

Reply:
Fixed in https://github.com/eclipse-ee4j/batch-tck/pull/40